### DISCUSSION TITLE
Tensorflow events log is too big

### DISCUSSION THREAD

**Discussion by Vadim2S** [voicepaw/so-vits-svc-fork#969] [Q&A]
### Describe the bug

I am train 6 voice corpus combined size of 10GB. Now I am at epoch 1700 and events.out.tfevents... file has size of 40GB. I am can not operate with such big file now. And I am wonder if my hard drive can withstand it at default config epoch 10000? :-)

### To Reproduce

Nothing special

### Version

4.1.7

### Platform

Ubuntu 20.04

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### No Duplicate

- [X] I have checked existing issues to avoid duplicates.

**Comment by 34j** [comment]
It works fine if you delete it. This repository is designed to support around like 100 epochs of training, but you can modify the code yourself to suit your needs.
